src/main/java/com/googleinterns/zoomtube/servlets/TranscriptServlet.java

```diff
@@ -57,7 +57,7 @@
   public static final String ATTR_DURATION = "dur";
   public static final String TAG_TEXT = "text";
 
-  private DatastoreService datastore;
+  private static DatastoreService datastore;
```

Comment:
I don't think you want this to be static.  A single instance might not be thread safe...

Reply:
You're also overwriting it in init, which may cause problems if the reference switches in the middle of a call in another thread

Reply:
Thanks for the info! done